aep/general/oas.yaml

```diff
@@ -0,0 +1,501 @@
+  /publishers:
+    get:
+      parameters:
+        - in: query
+          name: max_page_size
+          required: true
+          schema: {}
+          type: integer
+        - in: query
+          name: page_token
+          required: true
+          schema: {}
+          type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                properties:
+                  results:
+                    items:
+                      $ref: '#/components/schemas/publisher'
+                    type: array
+                type: object
+          description: Successful response
+    post:
+      parameters:
+        - in: query
+          name: id
+          required: true
+          schema: {}
+          type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/publisher'
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/publisher'
+          description: Successful response
+  /publishers/{publisher}:
+    delete:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+      responses:
+        '200':
+          content: null
+          description: ''
+    get:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/publisher'
+          description: Successful response
+    patch:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/publisher'
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/publisher'
+          description: Successful response
+    put:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
+          type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/publisher'
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/publisher'
+          description: Successful response
+  /publishers/{publisher}/books:
+    get:
+      parameters:
+        - in: path
+          name: publisher
+          required: true
+          schema: {}
```

Comment:
Is this needed for every path parameter?

Reply:
no it's a bug with aepc to include it - let me try to remove that.